templates/ts-apollo-kafka-mongo-backend/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+services:
+
+  zookeeper:
+    image: wurstmeister/zookeeper
+    ports:
+      - 2181:2181
+
+  kafka:
+    image: wurstmeister/kafka
+    ports:
+      - 9092:9092
+    links:
+      - zookeeper
+    environment:
+      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+      KAFKA_ADVERTISED_HOST_NAME: ${HOST_IP}
+      KAFKA_AUTO_CREATE_TOPICS_ENABLE: 'false'
```

Comment:
```suggestion
      KAFKA_AUTO_CREATE_TOPICS_ENABLE: 'true'
```
?

Reply:
Thanks, I had that in for testing fix in graphql-kafka-subscriptions